sbin/common/config_init.sh

```diff
@@ -544,6 +573,7 @@ function configDefaults() {
   BUILD_CONFIG[DOCKER_SOURCE_VOLUME_NAME]=${BUILD_CONFIG[DOCKER_SOURCE_VOLUME_NAME]:-"openjdk-source-volume"}
 
   BUILD_CONFIG[CONTAINER_NAME]=${BUILD_CONFIG[CONTAINER_NAME]:-openjdk_container}
+  BUILD_CONFIG[CONTAINER_IMAGE]=${BUILD_CONFIG[CONTAINER_IMAGE]:-"ubuntu:18.04"}
```

Comment:
Is 18.04 still within support? If not we should bump this baseline.

Reply:
Yes it should be ok until 2028: https://ubuntu.com/about/release-cycle
Anyway, I'm not going to be reponsible for default os, unless it is rhel/fedora, where I can moreover be sure what I'm doing (unless it is pcsc-lite-devel and libstdc++-static...)